template.yaml

```diff
@@ -118,6 +118,42 @@ Resources:
             Method: post
             RestApiId: !Ref NvaHandleServiceApi
 
+  NvaUpdateHandleFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub /aws/lambda/${NvaUpdateHandleFunction}
+      RetentionInDays: 180
+
+  NvaUpdateHandleFunction:
```

Comment:
CreateHandleHandler has not its own endpoint in template and openapi yet?

Reply:
it is in there, just not changed. CreateHandleHandler is not new, just moved and edited, but the way I did it, git did not see the move.